### PR TITLE
Fix bug 1210396: Add newsletter strings to bedrock.

### PR DESCRIPTION
**Pull request by pmclanahan**
I did it in the view rather than the template, but it works
just as well.

### CODE REVIEW

**Comment by pmclanahan** [comment]
@pascalchevrel this seems to work well, but some strings aren't translated when I run this locally that are on production. In my searching it seems that there are still strings in `newsletter.lang` that are being used in prod and haven't been moved to `mozorg/newsletters.lang`. Not sure if that's the deal or not, but since I remember `newsletter.lang` being removed from the default files recently I figured it may be relevant. Let me know what you think.

**Comment by pmclanahan** [comment]
@pascalchevrel bump

File: bedrock/newsletter/views.py [hunk 1]
```diff
@@ -51,6 +51,149 @@
...
+        'title': _lazy(u'Addon Development')},
+    u'addons': {
+        'description': _lazy(u''),
+        'title': _lazy(u'About Addons')},
+    u'affiliates': {
+        'description': _lazy(u'A monthly newsletter to keep you up to date with the '
+                             u'Firefox Affiliates program.'),
+        'title': _lazy(u'Firefox Affiliates')},
+    u'ambassadors': {
+        'description': _lazy(u'A monthly newsletter on how to get involved with Mozilla on your campus. '),
```
Comment (pascalchevrel) [inline bedrock/newsletter/views.py]: There is an extra trailing space on this string

**Comment by pascalchevrel** [comment]
I rechecked all the strings in your PR and I think the missing strings are actually those that are changed as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1199799

I don't know if you want to update your PR to reflect the changes asked in this bug or if you prefer a separate PR.

**Comment by pmclanahan** [comment]
@pascalchevrel I've updated this PR to change the names of those newsletters. Is that all that you needed?

**Comment by pascalchevrel** [comment]
yes  I think it's fine, sorry for the delay I was on PTO

**Pull request closed without merging**